pkg/machines/libvirt.es6

```diff
@@ -595,4 +593,25 @@ function buildConsoleVVFile(consoleDetail) {
         'fullscreen=0\n';
 }
 
+function doUsagePolling (name, connectionName) {
+    logDebug(`doUsagePolling(${name}, ${connectionName})`);
+
+    const canFailHandler = ({exception, data}) => {
+        console.info(`The 'virsh' command failed, as expected: "${JSON.stringify(exception)}", data: "${JSON.stringify(data)}"`);
+    };
+
+    return dispatch => {
```

Comment:
I think this can be `(dispatch, getState)` and then `doUsagePolling` can call `usagePollingEnabled`.